.env.example

```
@@ -0,0 +1,102 @@
+WAREHOUSE_ROOT=/app/Tazama_Hudi_warehouse
+# OZONE_S3_ENDPOINT=http://localhost:9878
+# OZONE_S3_ACCESS_KEY=admin
+# OZONE_S3_SECRET_KEY=admin
+
+# Logging
+LOG_LEVEL=INFO
+
+# AWS CLI Configuration
+
+AWS_DEFAULT_REGION=us-east-1
+AWS_BUCKET_NAME=tazama
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The AWS_BUCKET_NAME key should go before the AWS_DEFAULT_REGION key

(UnorderedKey)